l10n_pt_account/account_config.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+# Copyright 2010-2016 ThinkOpen Solutions
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)./
+from openerp import models, fields, api
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    income_move_account_id = fields.Many2one(
+        'account.account', string="Gain Account",
+        help="Account to register the differences "
+             "of values in invoices in gain (credit)")
+    expense_move_account_id = fields.Many2one(
+        'account.account', string="Loss Account",
+        help="Account to register the differences "
+             "of values in invoices in loss (debit)")
+
+
+class AccountPtConfig(models.Model):
+    _inherit = 'account.config.settings'
```

Comment:
@andreparames Isn't `account.config.settings` a wizard? It should rather be a `models.TransientModel` then.